qrscan/models.py

```diff
@@ -0,0 +1,123 @@
+import uuid
+from io import BytesIO
+from PIL import Image
+import qrcode
+import secrets
+
+from django.core.files import File
+from django.db import models
+
+from home.models import Allocation, Student
+
+
+def generate_secret_key():
+    return secrets.token_hex(32)
+
+
+class MessCard(models.Model):
+    id = models.UUIDField(
```

Comment:
check this https://github.com/pshrmn/notes/blob/master/django/models.md#primary-keys
maybe we might not need id

Reply:
It's important as anyone can generate an QR with a ID if its like Integer (1, 2, 3....)
Anyone can generate these numbers by guessing but by using this method nobody can do that

Reply:
lets discuss on this.